components/compliance-service/api/tests/08_search_profiles_spec.rb

```diff
@@ -180,6 +180,30 @@ def reporting;
     }.to_json
     assert_equal(expected_data, actual_data.to_json)
 
+    # Filter by status
+    actual_data = GRPC reporting, :list_profiles, Reporting::Query.new(
+        filters: [
+            Reporting::ListFilter.new(type: "end_time", values: ["2018-03-04T#{END_OF_DAY}"]),
+            Reporting::ListFilter.new(type: 'status', values: ['passed'])
+        ],
+        page: 1, per_page: 2)
+    expected_data = {
+        "profiles" => [
+            {
+                "name" => "nginx-baseline",
+                "title" => "DevSec Nginx Baseline",
+                "id" => "09adcbb3b9b3233d5de63cd98a5ba3e155b3aaeb66b5abed379f5fb1ff143988",
+                "version" => "2.1.0",
+                "status" => "passed"
+            }
+        ],
+        "counts" => {
+            "total" => 1,
+            "passed" => 1
+        }
+    }.to_json
+    assert_equal(expected_data, actual_data.to_json)
```

Comment:
We have a helper function for this kind of test: https://github.com/chef/automate/blob/53665d66aaeb9c4cebb1d54ce41fb16375665ae0/components/compliance-service/api/tests/run.rb#L105-L107

Reply:
ohh nice!